data/devices/glorious-model-d.device

```diff
@@ -0,0 +1,5 @@
+[Device]
+Name=SINOWEALTH Wired Gaming Mouse
```

Comment:
Why don't you set the name to `Glorious Model D` here?

Reply:
That's their vendor/product string, see [lsusb](https://github.com/enkore/gloriousctl/blob/latest/lsusb.txt#L8). I assume that's what's meant by "kernel name" in the example.

Reply:
There is no need to use the same name as shown in lsusb. I prefer to have names that better describe the device.